src/schema/city/index.ts

```diff
@@ -17,8 +16,13 @@ import EventStatus from "schema/input_fields/event_status"
 import cityData from "./city_data.json"
 import { pageable } from "relay-cursor-paging"
 import { connectionFromArraySlice } from "graphql-relay"
-import { LOCAL_DISCOVERY_RADIUS_KM } from "./constants"
+import {
+  LOCAL_DISCOVERY_RADIUS_KM,
+  NEAREST_CITY_THRESHOLD_KM,
+} from "./constants"
 import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import Near from "schema/input_fields/near"
```

Comment:
TIL… even though `Near` and `LatLngType` have largely the same shape, we can only use the one based on `GraphQLInputObjectType` (i.e. `Near`) as a field argument.